views/NEWUSERS.ejs

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<%- include('partials/head.ejs',{page:{name:'Welcome'}}) %>
+
+    
+    <title>Welcome to Critter</title>
+</head>
+<div class="container">
+    <div class="logo"></div>

```

Comment:
All of your indentation needs to be tabs, not spaces.

And make sure everything is at the correct tab depth.